src/bosh-softlayer-cpi/action/create_vm_test.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	. "bosh-softlayer-cpi/softlayer/common"
 	fakescommon "bosh-softlayer-cpi/softlayer/common/fakes"
```

Comment:
We only have to consider the case when the length is _exactly_ 64 characters long, VMs got created fine when its either > or < 64.